### DISCUSSION TITLE
An excess of email notifications of beta releases

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [Ulauncher/Ulauncher#1214] [General]
Today I had nine emails about beta releases. I have signed up, via GitHub, to receive release announcements from the repository. Yet: in a sense a beta is not a release; and the quantity of emails is undesirable. Thoughts?

**Comment by troycurtisjr** [comment]
Well Beta releases are certainly releases, it is in the name 😀. However, in this case, as you may have realized, these were the result of testing new automation to do the releases and is not indicative of a normal occurrences. Unfortunately, the only way to test the interaction with the releases API, is to interact with the releases API.

**Comment by LinuxOnTheDesktop** [comment]
I am glad to learn that the quantity of notifications is not normal!

**Reply by friday** [reply]
We switched away from Travis to GitHub Actions because [our old CI is not functional](https://travis-ci.community/t/exceeded-user-count-unlimited/10484). Gornostal has made the migration because he was the one to set this up originally and had the access to Launchpad.

Now I'm experimenting as well, because I want the releases to have a manual verification/change notes edit step, so it's not over unfortunately. Hopefully will be after today though.

**Reply by friday** [reply]
Done now. Shouldn't have to do this again (hopefully).
